src/app/beer_garden/requests.py

```diff
@@ -893,17 +893,24 @@ def handle_event(event):
             # the subset of fields that change "corrects" the parent
             existing_request = db.query_unique(Request, id=event.payload.id)
 
-            if existing_request:
-                for field in ("status", "output", "error_class", "status_updated_at"):
-                    setattr(existing_request, field, getattr(event.payload, field))
-
-                try:
-                    update_request(existing_request)
-                except RequestStatusTransitionError:
-                    pass
-            else:
+            if existing_request is None:
                 # Attempt to create the request, if it already exists then continue on
                 try:
                     db.create(event.payload)
                 except NotUniqueException:
                     pass
+            elif event.name != Events.REQUEST_CREATED.name:
```

Comment:
This just ensures we don't do any work on a "REQUEST_CREATED" event if we've already recorded the event locally.  The only thing that could have changed at that point is "status_updated_at", which isn't really useful yet.